job-runner/runner/runner.go

```diff
@@ -28,6 +28,14 @@ type Return struct {
 	Tries      uint // Number of tries this run, not including any previous tries
 }
 
+type Status struct {
+	Job       proto.Job
+	StartedAt time.Time
```

Comment:
would be good to add a comment that StartedAt here refers to when the entire Runner started, not necessarily this try of the job.

Reply:
Done.